### PR TITLE
NOISSUE - Charts for new core release 0.12.0

### PR DESCRIPTION
**Pull request by blokovi**
Signed-off-by: Ivan Milosevic <iva@blokovi.com>

### CODE REVIEW

File: charts/mainflux/templates/auth-deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+            - name: MF_AUTH_DB_USER
+              value: {{ index .Values "postgresql-auth" "postgresqlUsername" }}
+            - name: MF_AUTH_GRPC_PORT
+              value: "{{ .Values.auth.grpcPort }}"
+            - name: MF_AUTH_HTTP_PORT
+              value: "{{ .Values.auth.httpPort }}"
+            - name: MF_AUTH_LOG_LEVEL
+              value: {{ default .Values.defaults.logLevel .Values.users.logLevel }}
+            - name: MF_AUTH_SECRET
+              value: c8ee60b5-1317-4b7c-adb3-c0d73f30959c
```
Comment (nmarcetic) [inline charts/mainflux/templates/auth-deployment.yaml:41]: Whats this?

File: charts/mainflux/templates/notifier_smtp-deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+            - mountPath: /smtp-notifier.tmpl
+              name: email-notifier-config
+              subPath: email.tmpl              
+            - mountPath: /config.toml
+              name: email-notifier-config
+              subPath: config.toml                          
+          stdin: true
+          tty: true
+      imagePullSecrets:
+      - name: regcred
```
Comment (nmarcetic) [inline charts/mainflux/templates/notifier_smtp-deployment.yaml:107]: Image from private registry?

File: charts/mainflux/templates/auth-deployment.yaml (hunk 1 above)
Comment (blokovi) [reply charts/mainflux/templates/auth-deployment.yaml]: Looks like :)
I make it configurable from `values.yaml` and make it default `secret` same as in docker-compose.

File: charts/mainflux/templates/notifier_smtp-deployment.yaml (hunk 2 above)
Comment (blokovi) [reply charts/mainflux/templates/notifier_smtp-deployment.yaml]: Yes, left over from development.
Removed, thanks

**Review by manuio — APPROVED** [review]
LGTM

**Review by nmarcetic — APPROVED** [review]

**Pull request merged**